doc/changes/changes_4.0.0.md

```diff
@@ -22,4 +22,5 @@ The old variant is intentionally not supported anymore to tighten security.
 
 ## Plugin Updates
 
+* Added `project-keeper-maven-plugin:0.4.0`
```

Comment:
Group id is missing (see https://github.com/exasol/release-droid/blob/main/doc/user_guide/changes_file_template.md)

Reply:
Fixed.